### PR TITLE
Creating the basic structure in the index file

### PR DESCRIPTION
**Pull request by ghost**
@fxisco I just started but what do you think? next, I will do a little of CSS to arrange everything. Honestly, I didn´t exactly know what to do.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Slide Show</title>
+		<style>
+		body{
+
+		}
+		.slideShow-container{
```
Comment (fxisco) [inline index.html]: Could we separate the styles in other file?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+			background-color: rgba(255,255,255,0.3);
```
Comment (fxisco) [inline index.html]: Good practice is to alphasort css attribute

File: index.html [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+			position: absolute;
+			height: 95%;
+			width: 100%;
+			top: 10%;
+			left: 0%;
+		}
+		.slides-container{
+			position: absolute;
+			height: 100%;
+    		width: 100%;
```
Comment (fxisco) [inline index.html]: fix indentation. I personally recommend to use 2 spaces

File: index.html [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+    		position: absolute;
+    		top: 0;
+    		height: 100%;
+    		width: 4%;
+    		cursor: pointer;
+    		opacity: 0.3;
+		}
+		.Option-bar{
+			background-color: rgba(0,0,0,0.5);
+			position: absolute;
```
Comment (fxisco) [inline index.html]: You could group common classes that use the position absolute attribute

File: index.html [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+			top: 94%;
+			height: 6%;
+			width: 100%;
+		}
+		.play{
```
Comment (fxisco) [inline index.html]: Awesome use of classes 👍

File: index.html [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+			bottom: 6%;
+			visibility: hidden;
+		}
+		.title{
+		    height: 6%;
+		    position: absolute;
+		    top: 0%;
+		    background-color: rgba(255,255,255,69%);
+		    text-align: center;
+		    /* padding-top: 0%; */
```
Comment (fxisco) [inline index.html]: maybe you want to remove comments

File: index.html [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+			font-size: 255%;
+		}
+		.imgs{
+			position: absolute;
+			height: 100%;
+			overflow: hidden;
+			visibility: hidden;
+			width: 100%
+		}
+		.imgs img{
```
Comment (fxisco) [inline index.html]: why use `.imgs img` or maybe you could use direct child selector `>` to be more precise

File: index.html [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+			height:	189px;
+			opacity: 0.6;
+			cursor: pointer;
+		}
+
+		
+	</style>
+</head>
+<body>
+	<h2>SlideShow</h2>
```
Comment (fxisco) [inline index.html]: Every page should have a h1 for accessibility sake. Maybe you want to chage it to h1

**Review by fxisco — COMMENTED** [review]
- add the lang attribute to the html element

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: Yes we could, I was thinking on do it later, but I will do it now

File: index.html (hunk 2 above)
Comment (ghost) [reply index.html]: Thanks

File: index.html (hunk 3 above)
Comment (ghost) [reply index.html]: I did it, on the other commit

File: index.html (hunk 8 above)
Comment (ghost) [reply index.html]: I didn't know, I'll keep it in mind from now on.

File: index.html (hunk 6 above)
Comment (ghost) [reply index.html]: Sorry I did it after the commit

File: index.html (hunk 7 above)
Comment (ghost) [reply index.html]: I was thinking on using it, but didn't remember at the moment and the forget to do it.

File: index.html (hunk 4 above)
Comment (ghost) [reply index.html]: **How? What do you mean?**

**Pull request closed without merging**